src/Tests/Microsoft.NET.Sdk.WorkloadManifestReader.Tests/SdkDirectoryWorkloadManifestProviderTests.cs

```diff
@@ -31,6 +31,48 @@ void Initialize(string featureBand = "5.0.100", [CallerMemberName] string? testN
             Directory.CreateDirectory(_manifestVersionBandDirectory);
         }
 
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public void ItShouldReturnTheWorkloadVersion(bool useWorkloadSet)
+        {
+            Initialize();
+
+            CreateMockManifest(_manifestRoot, "8.0.100", "ios", "11.0.2", true);
+            CreateMockManifest(_manifestRoot, "8.0.200", "android", "33.0.2-rc.1", true);
+            CreateMockManifest(_manifestRoot, "8.0.200-rc.2", "maui", "15.0.1-rc.456", true);
+
+            if (useWorkloadSet)
+            {
+                CreateMockWorkloadSet(_manifestRoot, "8.0.200", "8.0.200", @"
```

Comment:
I think we should change the workload set version to something that is different than the feature band version (such as 8.0.202), to help verify the right version is being returned.